packages/compass-import-export/src/components/export-modal/export-modal.jsx

```diff
@@ -84,6 +84,7 @@ class ExportModal extends PureComponent {
     toggleFullCollection: PropTypes.func.isRequired,
     selectExportFileType: PropTypes.func.isRequired,
     selectExportFileName: PropTypes.func.isRequired,
+    aggregation: PropTypes.object,
```

Comment:
in state we set it as `false` by default. maybe we can make it optional there?

Reply:
Changing it to default `{}` for better consistency

Reply:
in that case `!!this.props.aggregation` won't return false if its empty